redaxo/src/core/lib/be/controller.php

```diff
@@ -459,6 +459,6 @@ private static function includePath($path, array $context = [])
         if (isset($matches[2])) {
             $package = $package->getPlugin($matches[2]);
         }
-        return $package->includeFile(str_replace($package->getPath(), '', $path), $context);
+        return $package->includeFile(str_replace($package->getPath(), '', $path));
```

Comment:
Hmm warum gibts den error lt. Phpstan im bugfix branch jicht?

Reply:
Hmmm dieser parameter existiert im interface nicht, aber in 2 implementierungen.. @gharlan  ?

Reply:
Die wurden nachträglich hinzugefügt, im 5.1.
Entweder im Interface vergessen, oder Absicht aus BC-Gründen um das Interface nicht zu ändern.